sdk/eventhub/Azure.Messaging.EventHubs/src/TrackOneClient/EventDataDiagnosticExtensions.cs

```diff
@@ -65,7 +65,7 @@ internal static class EventDataDiagnosticExtensions
         /// 
         /// </example>
 
-        public static Activity ExtractActivity(this EventData eventData, string activityName = null)
+        internal static Activity ExtractActivity(this EventData eventData, string activityName = null)
```

Comment:
Doesn't really need to be changed; the class is internal which limits the visibility of the members.  Typically, if a class is non-public, I normally make each of the members public just to remove any doubt about it being available.    I'm good either way for this, just thought that I'd mention it.

Reply:
I changed it because for some reason it was not allowing me to use an extension method when its access modifier is less restrictive than the one in `TrackOne.EventData` (internal). It seems changing it back to `public` still works, so I'll revert it. I'm not really sure what I was doing differently before.

Reply:
![](https://media2.giphy.com/media/fMvvwdTWamlA4/giphy.gif)

Computers, man.... they make no sense.